content/millennium/dstu2/diagnostic/diagnostic-report.md

```diff
@@ -26,7 +26,8 @@ The following fields are returned if valued:
 * [Issued date/time](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.issued){:target="_blank"}
 * [Performer (practitioner)](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.performer){:target="_blank"}
 * [Request (order)](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.request){:target="_blank"}
-* [ContentType and URL (fully qualified link to the document](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.presentedForm){:target="_blank"}
+* PresentedForms:
+  * [ContentType, URL, and Title (fully qualified link to the document)](http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport.presentedForm){:target="_blank"}
```

Comment:
In R4, we've been breaking up subfields individually. [Example for DocRef](http://fhir.cerner.com/millennium/r4/documents/document-reference/). Do we want to do that for dstu2 as well?

@MaxPhilips

Reply:
i PMed adam asking for this :)

Reply:
[Update in this commit](https://github.com/cerner/fhir.cerner.com/pull/338/commits/0992908621c8ebd4207546d8463af9253f5968c1#diff-663275837ef95b5d96d65014e7c0e943R29-R32)